test/house_test.rb

```diff
@@ -0,0 +1,75 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/room'
+require './lib/house'
+
+class HouseTest < Minitest::Test
+def test_it_exists
+  house = House.new("$400000", "123 sugar lane")
+  assert_instance_of House, house
+end
+
+def test_it_has_price
+  house = House.new("$400000", "123 sugar lane")
+  assert_equal "$400000", house.price
```

Comment:
Keep a *close* eye on the interaction pattern; we wanted this method to return an integer